docs/SettingUp.md

```diff
@@ -52,11 +52,5 @@ If you plan to use Intellij IDEA (highly recommended):
 
 1. **Learn the design**
 
-   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [AddressBook’s architecture](DeveloperGuide.md#architecture).
+   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [UGTeach’s architecture](DeveloperGuide.md#architecture).
 
-1. **Do the tutorials**
-   These tutorials will help you get acquainted with the codebase.
-
-   * [Tracing code](tutorials/TracingCode.md)
-   * [Adding a new command](tutorials/AddRemark.md)
-   * [Removing fields](tutorials/RemovingFields.md)
```

Comment:
Not sure if these really need to be deleted.

Reply:
[If you click into these, it actually throws an error, even in the AB3 DG. see here](https://se-education.org/addressbook-level3/SettingUp.html)

Reply:
Alright then makes sense to delete. Thanks for catching it.